sonarqube/license.md

```diff
@@ -1 +1 @@
-View [license information](http://www.gnu.org/licenses/lgpl.txt) for the software contained in this image.
+SonarQube Community Edition is licensed under [GNU Lesser General Public License, Version 3.0]((http://www.gnu.org/licenses/lgpl.txt)). SonarQube Developer, Enterprise and Data Center Editions are licensed under [SonarSource Terms and Conditions](https://www.sonarsource.com/docs/sonarsource_terms_and_conditions.pdf)
```

Comment:
```suggestion
SonarQube Community Edition is licensed under [GNU Lesser General Public License, Version 3.0]((http://www.gnu.org/licenses/lgpl.txt)). SonarQube Developer, Enterprise, and Data Center Editions are licensed under [SonarSource Terms and Conditions](https://www.sonarsource.com/docs/sonarsource_terms_and_conditions.pdf)
```

Reply:
added to the PR, thanks for helping with my grammar :+1: